### PR TITLE
pt-br-translation: translated small files

### PR DESCRIPTION
**Pull request by jhonmike**
Updated with the master branch's English doc.

It is translated small files with only title and description.

### CODE REVIEW

File: content/pt-br/docs/ops/configuration/telemetry/_index.md [hunk 1]
```diff
@@ -1,8 +1,6 @@
 ---
-title: Observability
-description: Helps you manage telemetry collection and visualization in a running mesh.
+title: Observabilidade
+description: Ajuda a gerenciar a coleção e a visualização de telemetria no mesh em execução
```
Comment (jhonmike) [inline content/pt-br/docs/ops/configuration/telemetry/_index.md]:
```suggestion
description: Ajuda a gerenciar a coleção e a visualização de telemetria no mesh em execução.
```

**Comment by jhonmike** [comment]
review please!

**Comment by jhonmike** [comment]
ping

**Comment by jussarasoares** [comment]
@googlebot I fixed it.

**Comment by jhonmike** [comment]
Thanks @jussarasoares, about the files that have not been translated, the reason is that it was an update of the English version in the files that are in the pt-br directory

**Comment by ericvn** [comment]
I did kick off the tests. You'll still need to fix the CLA and someone as an Infrastructure maintainer will need to approve.

**Comment by jhonmike** [comment]
@googlebot I fixed it.

**Comment by jhonmike** [comment]
I am separating this PR, one with the updates of the files with the official documentation and then I will send another PR with the translations.

Updating with English version: https://github.com/istio/istio.io/pull/6428

**Pull request closed without merging**